.github/workflows/ci.yaml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build:
     name: Build Snap
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
IMO, the `runs-on` change is not really needed, as it just affects the CI environment. The snap build and deploy will still make use of core24 regardless of github runner env.

Reply:
Yes, that is correct, but `ubuntu-latest` is running 22.04 LTS and building the snap with 24.04 makes it use  dependencies of a version higher than that on 22.04. This causes one of the tests to fail.